### ISSUE TITLE
How to add custom source[which we are getting Elastic search or excel sheet]  in PnP search results

### ISSUE THREAD

**Issue by rameshbaburanga** [microsoft-search/pnp-modern-search#1573]
How to add custom source[which we are getting Elastic search or excel sheet]  in PnP search results

**Comment by FranckyC** [comment]
Hi @rameshbaburanga, this is not supported in the open source version. Try this instead https://aequos-solutions.github.io/modern-data-visualizer/extensibility/custom_data_source/

**Comment by rameshbaburanga** [comment]
I understand that PnP search supports the source only SharePoint list/library data

**Comment by FranckyC** [comment]
We only support SharePoint search and Microsoft search. No other sources are planned.

**Comment by rameshbaburanga** [comment]
Hi @FranckyC ,
Thank you for providing help and support.

As you suggested above, we configured data visualizer webpart but we are unable to see the source of "My custom source" 
Attached the screen shot for reference.  Below version we installed in our site.
![image](https://user-images.githubusercontent.com/91315864/144362979-6f144610-b59d-48dd-80d3-6e8409a621a6.png)


![image](https://user-images.githubusercontent.com/91315864/144362884-177c1f59-95cc-4f79-9e77-50108e720e08.png)

Kindly help us here.
